pkg/wrap/wrap.go

```diff
@@ -0,0 +1,167 @@
+package wrap
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
+
+// ServerToClient returns a grpc.ClientConnInterface that can service all methods described by desc.
+// srv is the gRPC server implementation type (a type that could be passed to a grpc RegisterXxxServer function).
+//
+// Unary and streaming calls are supported.
+//
+// The call options grpc.Header and grpc.Trailer are supported for unary RPCs. All other call options are ignored.
+func ServerToClient(desc grpc.ServiceDesc, srv any) grpc.ClientConnInterface {
+	// check that srv is the right type to be a server for desc
+	expectType := reflect.TypeOf(desc.HandlerType).Elem()
+	if !reflect.TypeOf(srv).Implements(expectType) {
+		panic(fmt.Sprintf("grpcdynamic: srv must be of type %v", expectType))
```

Comment:
Where does this panic prefix come from? Shouldn't it be `wrap` or something?

Reply:
It's a leftover from being moved. It should indeed be `wrap`